python3/dnf_plugins/ptoken.py

```diff
@@ -0,0 +1,18 @@
+"""dnf plugin to add ptoken for repos"""
+import dnf
+
+class Ptoken(dnf.Plugin):
+    """ptoken plugin class"""
+    name = "ptoken"
```

Comment:
It seems this "name" is required by dnf plugin?
(Would be better to keep a consistent formating style for both plugins, e.g. the blank lines here.)

Reply:
Yes, it just follow the dnf plugin protocols, find this in the comments:
refer to https://dnf.readthedocs.io/en/latest/api_plugins.html